packages/fxa-auth-server/lib/routes/subscriptions.js

```diff
@@ -368,13 +376,15 @@ class DirectStripeRoutes {
           canceled_at,
           plan: { product: productId },
         } = subscription;
-        activeSubscriptions.push({
-          uid,
-          subscriptionId,
-          productId,
-          createdAt: created * 1000,
-          cancelledAt: canceled_at ? canceled_at * 1000 : null,
-        });
+        if (['trialing', 'active', 'past_due'].includes(subscription.status)) {
```

Comment:
Just double-checking: do we allow access to products when subscriptions are in a `past_due` state?